cmd/dcrtime_fsck/dcrtime_fsck.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/decred/dcrd/chaincfg"
+	"github.com/decred/dcrd/dcrutil"
+	"github.com/decred/dcrtime/dcrtimed/backend"
+	"github.com/decred/dcrtime/dcrtimed/backend/filesystem"
+)
+
+var (
+	defaultHomeDir = dcrutil.AppDataDir("dcrtimed", false)
+
+	file        = flag.String("file", "", "journal of modifications if used (will be written despite -fix)")
+	fix         = flag.Bool("fix", false, "Try to correct correctable failures")
+	dcrtimeHost = flag.String("host", "", "dcrtime block explorer")
```

Comment:
rename to something better, such as dcrBlockExplorer perhaps.

Reply:
call it dcrdataHost since it uses the dcrdata api (and would not work with an insight block explorer)